### DISCUSSION TITLE
Old Man need HELP....

### DISCUSSION THREAD

**Discussion by LettisGithub** [roleoroleo/yi-hack-Allwinner#437] [Q&A]
Hi roleoroleo and all the other famous programming heroes, first of all my respect for your brilliant performance.
Now to what I would like...I have several Yi cams, Yi home 1080p_5x,Yi Dome 1080p_4x,Yi outdoor 1080p_4x.I brickt
3x home cam and 2x dome cam why??? Because I'm to stupid, to old. 2x home, 1x dome 3x outdoor work but... they record on the micro SD ok, but I want them with the Nas on mine Router (Fritz.Box)
connect to record and I can't get it to work... I've been trying for 2 weeks now and I've already got sore fingers from typing.
For example, I don't know where to write which password and user, I've already tried all the variants. I have enabled all ports for the cams in the router to choose themselves, no success
Do you change the settings in the config file? To do this, I take the card out of the (switched off) cam and plug it into a micro SD adapter in my computer and open the config with Notepad++ and change the data,
Maybe that's a mistake but I don't know how else to do it, my English is very bad and I have everything translated with Google. I'll attach the config here, please be kind and help me.
With special regards Wolfgang

HTTPD=yes
TELNETD=no
SSHD=yes
FTPD=yes
BUSYBOX_FTPD=yes
DISABLE_CLOUD=yes
REC_WITHOUT_CLOUD=yes
MQTT=no
RTSP=no
RTSP_STREAM=high
RTSP_AUDIO=no
ONVIF=no
ONVIF_WSDD=no
ONVIF_PROFILE=high
ONVIF_NETIF=wlan0
TIME_OSD=yes
NTPD=no
NTP_SERVER=pool.ntp.org
PROXYCHAINSNG=no
SWAP_FILE=yes
RTSP_PORT=554
HTTPD_PORT=80
USERNAME=..ttisNas
PASSWORD=.....aud!!
TIMEZONE=CET-1CEST,M3.5.0,M10.5.0/3
FREE_SPACE=15
FTP_UPLOAD=yes
FTP_HOST=Fritz.Box
FTP_DIR=http://192.168.178.1/nas#/files/KINGSTON/Kamera
FTP_DIR_TREE=no
FTP_USERNAME=..ttisNas
FTP_PASSWORD=....aud!!
FTP_FILE_DELETE_AFTER_UPLOAD=yes
CHECK_UPDATES=no
SSH_PASSWORD=....aud!!
CRONTAB=
RTSP_ALT=no
SPEAKER_AUDIO=no
SNAPSHOT=yes
SNAPSHOT_VIDEO=no
SNAPSHOT_LOW=yes
TIMELAPSE=no
TIMELAPSE_FTP=no
TIMELAPSE_DT=60
TIMELAPSE_VDT=
DEBUG_LOG=yes  

This is in my router

All devices connected to the FRITZ!Box are protected from unwanted access from the Internet. However, some applications, such as online games, must be accessible to other Internet participants. You can allow such connections by setting up port sharing.

       Device/name          IP               Releases     Independent      port release     Port assigned externally IPv4     Port assigned externally IPv6
         YiDome3       192.168.178.64      21                0 active              21                                                     
                             
         YiHome5       192.168.178.61      21                0 active              21

-------------------------------------------------- -------------------------------------------------- -------
Maybe you could send me an Email?My EMail is: W.E.deletter@t-online.de
Thanks

**Comment by roleoroleo** [comment]
I don't understand exactly your goal.
But...
You can change the configuration of the cam (or at least, the portion related to this hack) opening the web gui with a browser.
Check the IP address of the cam and use your preferred browser.
About the records, you can't save them directly to your server (or storage connected to the router). But you can copy the mp4 files to a FTP server. If your router is able to publish a FTP server, inside this page http://IP_OF_YOUR_CAM/?page=eventsdir you can configure the address of the server, and the credetials to access it (username and password).

**Reply by LettisGithub** [reply]
Hey roleroroleo,thanks for your answer,so i have done...FTP_DIR=http://192.168.178.1/nas#/files/KINGSTON/Kamera....but no files got on the Nas from the cam,so thats my hole problem.my question ,must i allow the cam`s to go to the Internet? I`m confused,sorry man...

**Reply by roleoroleo** [reply]
No, you don't need to enable internet access in the cam.
But I think that your configuration is not ok.
Here an example of my configuration:
![immagine](https://github.com/user-attachments/assets/a7a422c3-6ea6-4b64-af7c-39fcc437ce69)

**Reply by LettisGithub** [reply]
Thank you .The username and Pass are 2 times in the Config,what is for what?im confused,sorry Roleoroleo,would you send me a picture of the whole config that runs?again sorry my friend,my brain is to old.
spezial regards

**Comment by roleoroleo** [comment]
This is my configuration:
```
root@yi-hack-allwinner:~# cat /home/yi-hack/etc/system.conf
HTTPD=yes
TELNETD=yes
SSHD=yes
FTPD=yes
BUSYBOX_FTPD=no
DISABLE_CLOUD=yes
REC_WITHOUT_CLOUD=yes
MQTT=yes
RTSP=yes
RTSP_ALT=no
RTSP_STREAM=both
RTSP_AUDIO=aac
SPEAKER_AUDIO=yes
SNAPSHOT=yes
SNAPSHOT_VIDEO=yes
SNAPSHOT_LOW=no
ONVIF=yes
ONVIF_WSDD=yes
ONVIF_PROFILE=both
ONVIF_WM_SNAPSHOT=yes
TIME_OSD=yes
NTPD=yes
NTP_SERVER=192.168.1.10
PROXYCHAINSNG=no
SWAP_FILE=yes
KERNEL_TUNING=yes
RTSP_PORT=554
ONVIF_PORT=80
HTTPD_PORT=80
USERNAME=
PASSWORD=
TIMEZONE=CET-1CEST,M3.5.0,M10.5.0/3
EVENTS_TIME=autodetect
FREE_SPACE=30
FTP_UPLOAD=yes
FTP_HOST=192.168.1.10
FTP_DIR=/camera/
FTP_DIR_TREE=yes
FTP_USERNAME=roleo
FTP_PASSWORD=*****
FTP_FILE_DELETE_AFTER_UPLOAD=no
SSH_PASSWORD=
CRONTAB=
TIMELAPSE=yes
TIMELAPSE_FTP=no
TIMELAPSE_DT=60
TIMELAPSE_VDT=0 0 * * *
DEBUG_LOG=no
TIMELAPSE_FTP_SAME_NAME=no
SWAP_SWAPPINESS=15
STATIC_IP=
STATIC_MASK=
STATIC_GW=
STATIC_DNS1=
STATIC_DNS2=
ONVIF_AUDIO_BC=G711
ONVIF_FAULT_IF_UNKNOWN=no
ONVIF_SYNOLOGY_NVR=no
```

**Comment by LettisGithub** [comment]
Thanks a lot,i think this could help..........
